src/components/Config.tsx

```diff
@@ -163,16 +164,23 @@ const YamlNodeWithDescription = ({
 
   return (
     <div
-      style={{ paddingLeft: `${root ? 0 : INDENT_SIZE}px` }}
+      style={{ paddingLeft: `${root ? 0 : INDENT_SIZE}px` }} // TODO: dynamic class
```

Comment:
considering `INDENT_SIZE` is a constant, this can be safely extracted into a conditional class